Gunta/Ejecicios/ejercicios_uba/funciones/ejercicio11.py

```diff
@@ -0,0 +1,38 @@
+"""Contar la cantidad de letras de un telegrama que termina en punto sin utilizar
+funciones de string, salvo la que indica la longitud"""
+
+
+def contar_telegrama(telegrama):
+    letras = "abcdefghijklmnñopqrstuvwxyz"
+    contador_letras = 0
+    i = 0
+    #corte = 0
+    while i < len(telegrama): #and corte != 1:
```

Comment:
Cuando pushees, mandá sólo la versión final andando. Idealmente sin comentarios pero si tenés otras alternativas y las querés dejar en el mismo archivo porque te sirven, comentalas. Este while no va. Va el for que viene después.